### ISSUE TITLE
Not Accessible to Screen Readers

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/shopping-list#4]
The provided code and the completed example are not accessible to screen reader users. There is no way for a screen reader user to know if an item is checked or not or what item a check or delete button is associated with.

VoiceOver screencast:
http://j4p.us/1s2z00240O39

Working on a PR that solves this implicitly using semantics and progressive enhancement.
